demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -240,7 +241,41 @@ def verify_readme_image_paths(self) -> bool:
             return False
         return True
 
-    def check_readme_relative_image_paths(self, is_pack_readme: bool = False) -> list:
+    @error_codes('RM112')
+    def check_readme_relative_url_paths(self, is_pack_readme: bool = False) -> list:
+        """ Validate readme url relative paths.
+            prints an error if relative paths in README are found since they are not supported.
+
+        Arguments:
+            is_pack_readme (bool) - True if the the README file is a pack README, default: False
+
+        Returns:
+            list: List of the errors found
+        """
+        error_list = []
+        # If error was found, print it only if its not a pack readme. For pack readme, the PackUniqueFilesValidator
+        # class handles the errors and printing.
+        should_print_error = not is_pack_readme
+        relative_urls = re.findall(r'^(?![!])(\[.*?\])\(((?![http|#|mailto:]).*?)\)$', self.readme_content,
```

Comment:
this regex seems not to work. Check this case
[Good link](hello.co.il)
`^(?![!])(\[.*?\])\(((?!(?:https?:\/\/)|#|(?:mailto:)).*?)\)$`